platforms/opencl/src/OpenCLNonbondedUtilities.cpp

```diff
@@ -379,6 +383,9 @@ void OpenCLNonbondedUtilities::prepareInteractions(int forceGroups) {
     // Segment the command stream to avoid stalls later.
     if (groupKernels[forceGroups].hasForces)
         context.getQueue().flush();
+    #else
+    if (isAMD)
```

Comment:
I thought you wanted only a single flush before computeNonbonded?  You added a second one here.

Reply:
I unintentionally had two flushes in while making the `isAMD` change and I was surprised to see that pme went from 400 -> 440 ns/day. I can remove it but this is perm 4 and that previously gave good results on [5700 XT on Linux](https://github.com/openmm/openmm/issues/3937#issuecomment-1416377734). Would you like to benchmark it first?

Reply:
I don't observe any benefit to pme from including the second flush on Linux.  But it also doesn't hurt, so if it helps on Windows, it's fine to include it.

Reply:
Let's keep it then! I moved the check out of the Mac section since that OS + GPU combination doesn't exist.